packages/grafana-ui/src/components/Table/RowsList.tsx

```diff
@@ -333,3 +377,45 @@ export const RowsList = (props: RowsListProps) => {
     </>
   );
 };
+
+/**
+ * Calculate an esimated bounding box for a block
+ * of text using an offscreen canvas.
+ *
+ * TODO: Move this 🚚
+ */
+function getTextBoundingBox(
+  text: string,
+  headerGroups: HeaderGroup[],
+  osContext: OffscreenCanvasRenderingContext2D | null,
+  lineHeight: number,
+  defaultRowHeight: number
+) {
+  let width = 300;
+  if (typeof headerGroups[0].headers[1].width === 'number') {
```

Comment:
might be easier to read if `headerGroups[0].headers[1]` was pulled into a `const header = ` above

Reply:
this whole if/else block can probably be replaced with `Number(headerGroups[0].headers[1])`

Reply:
Oops, and this should probably be updated to grab the correct header as well 😆

Reply:
OK, simplified this down 😄